src/hooks/disputable-voting-logic.js

```diff
@@ -10,6 +11,50 @@ function useGetVotes() {
   return votes
 }
 
+export function useGetVote(voteId) {
+  const [loading, setLoading] = useState(false)
+  const [extendedProperties, setExtendedProperties] = useState(null)
+  const [disputableVoting] = useApp('disputable-voting')
+  const [vote] = useDisputableVoting(disputableVoting, (app) => {
+    return app.vote(voteId)
+  })
+
+  useEffect(() => {
+    let cancelled = false
+
+    async function getVoteSettings() {
+      setLoading(true)
+
+      if (vote) {
+        const collateral = await vote.collateralRequirement()
+
+        const extendedProperties = {
+          settings: await vote.setting(),
+          status: await vote.status(),
+          collateral: collateral,
+          token: await collateral.token(),
+        }
+
+        if (!cancelled) {
+          setExtendedProperties(extendedProperties)
+          setLoading(false)
+        }
+      }
+    }
```

Comment:
Just thinking about error handling here, the loading status and whether we can make some of these requests concurrently. I wasn't able to get this branch loading correctly so will come back to this in a bit after we've caught up! :)